components/concept/storage/src/main/java/mozilla/components/concept/storage/History.kt

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.storage
+
+/**
+ * An interface which defines read/write methods for History data.
+ */
+interface History {
```

Comment:
Should this be called `HistoryStorage`. Confused me at first. The instances are all called `historyStorage`.